### PR TITLE
[Improvement-#7529][tools] Init DB schema from the full sql file.

### PR DESCRIPTION
**Pull request by lgcareer**
## Purpose of the pull request

Improvement the issue #7529 ,Init DB schema from the full sql file.

## Brief change log
Change the DolphinSchedulerManager、CreateDolphinScheduler and UpgradeDao
Modify the file name of dolphinscheduler_postgre.sql to dolphinscheduler_postgresql.sql

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by caishunfeng — COMMENTED** [review]
LGTM

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java [hunk 1]
```diff
@@ -42,10 +42,14 @@ public static void main(String[] args) {
 
         @Override
         public void run(String... args) throws Exception {
+            if (dolphinSchedulerManager.schemaIsInitialized()) {
+                dolphinSchedulerManager.upgradeDolphinScheduler();
+                logger.info("upgrade DolphinScheduler finished");
+            } else {
+                dolphinSchedulerManager.initDolphinScheduler();
+            }
             dolphinSchedulerManager.initDolphinScheduler();
```
Comment (kezhenxu94) [inline dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java]: it's initialized twice

```suggestion
```

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java [hunk 2]
```diff
@@ -42,10 +42,14 @@ public static void main(String[] args) {
...
             logger.info("init DolphinScheduler finished");
```
Comment (kezhenxu94) [inline dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java]: What about also moving this into the `else` block?

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java (hunk 1 above)
Comment (lgcareer) [reply dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java]: Thanks,you are right,I will remove it.

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java (hunk 2 above)
Comment (lgcareer) [reply dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java:52]: Great,I refetched the dev branch,DolphinSchedulerManager class can't import the UpgradeDao.
image

Comment (lgcareer) [reply dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java]: Great.Done.

File: dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java (hunk 1 above)
Comment (lgcareer) [reply dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/datasource/CreateDolphinScheduler.java]: Hi,I have done. please recheck it ,thx

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**